### ISSUE TITLE
small thing

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/react-only-13#5]
during your learning - you'll also implement this page into react: https://tympanus.net/Blueprints/OnScrollEffectLayout/

**Comment by khyati0212** [comment]
@atherdon What is expected output of this task? Do I need to create same page given in example?

**Comment by atherdon** [comment]
yes, you'll create a separated page, with components. so right now it's
HTML just, but after your work it'll be a page inside react project

**Comment by khyati0212** [comment]
Ok

**Comment by atherdon** [comment]
it'll also show you an approach of converting HTML into react components. like another practice. and it also looks cool
